src/components/Button/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import polished from "polished";
```

Comment:
React, Component and polished are unused

Reply:
Polished is used under the hood - this is the problem I guess, since we use `lighten` function for hover effects, specified in theme. But in theme it's only a string which will be evaluated in actual components - so we have to import `polished`. There has to be a better way.

You are right about React of course here.